### DISCUSSION TITLE
zh/guides/coral-edge-tpu-on-raspberry-pi/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#14547] [Docs]
# zh/guides/coral-edge-tpu-on-raspberry-pi/

通过Ultralytics YOLOv8 了解如何使用 Coral EdgeTPU 提升 Raspberry Pi 的 ML 性能。请遵循我们的详细设置和安装指南。

https://docs.ultralytics.com/zh/guides/coral-edge-tpu-on-raspberry-pi/?h=coral

**Comment by LJ-Hao** [comment]
Hi, I have transform yolov8s.pt model to edgetup format with int8=True, I get so many models, but what is the difference between int8, full_integer_quant and interger_quant?

**Reply by glenn-jocher** [reply]
@LJ-Hao the terms "int8," "full_integer_quant," and "integer_quant" refer to different quantization methods used to optimize models for Edge TPU. "int8" quantization converts model weights and activations to 8-bit integers, reducing model size and improving inference speed. "full_integer_quant" ensures all operations, including inputs and outputs, are quantized to integers, making the model fully compatible with Edge TPU. "integer_quant" typically refers to partial quantization where only certain parts of the model are quantized. For more details, visit the [Ultralytics documentation](https://docs.ultralytics.com/guides/coral-edge-tpu-on-raspberry-pi/).
